ads/inabox/inabox-host.md

```diff
@@ -14,4 +14,87 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-This is still in development. Check [here](https://github.com/ampproject/amphtml/issues/5700) for details. 
\ No newline at end of file
+# SDK integration for AMPHTML ad
+One goal of AMPHTML ad is for advertisers to create ad once and use 
+every where. This guide provides details about how an ad network can 
+join us to serve and render AMPHTML ad in various environment, for 
+example regular web & native mobile apps.
+
+## Terminologies
+
+###### AMPHTML ad
+Ad creatives that are written following the
+ [AMPHTML ad spec](https://amp.dev/documentation/guides-and-tutorials/learn/a4a_spec).
+ 
+###### inabox
+A term describes the situation that an AMPHTML ad is being served
+on a non-AMP environment, e.g a regular web page inside an iframe embed 
+or a native mobile app inside a WebView. This is very different from how
+ AMPHTML ad gets rendered on an AMP page, where the page and ad shares 
+ the same JS runtime.
+
+###### host
+The web page or native mobile app that the ad is being served on.
+
+###### friendly iframe
+An iframe on the host page that has the same origin, so that it has
+full access to the host page content.
+
+###### foreign iframe
+An iframe on the host page that has different origin, so it can only
+communicate to the host page via postMessage API.
+
+## Web SDK (ads tag) integration
```

Comment:
I don't think anyone calls this a "web sdk"?

Reply:
how about JS SDK?